convert/jenkinsjson/json/s3upload_test.go

```diff
@@ -0,0 +1,177 @@
+package json
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"testing"
+
+	harness "github.com/drone/spec/dist/go"
+	"github.com/google/go-cmp/cmp"
+)
+
+type s3runner struct {
```

Comment:
Pull from master and use type and function from ```base_test.go``` instead of duplicating them.

Reply:
ok taken care in this latest PR update.